flixel/text/FlxInputText.hx

```diff
@@ -19,6 +19,8 @@ class FlxInputText extends FlxText implements IFlxInputText
 	public var hasFocus(default, set):Bool = false;
 	
 	public var maxLength(default, set):Int = 0;
+
+	public var multiline(get, set):Bool;
```

Comment:
So I was wrong, it's not in FLxText, but it should be... or at last input text shouldn't be different from FlxText in this regard.

Reply:
Should I make a PR to add it to FlxText, or can you do it instead?

Reply:
not sure when i'll get to it, but i can